src/m365/spo/commands/tenant/tenant-settings-set.ts

```diff
@@ -637,6 +1137,15 @@ class SpoTenantSettingsSetCommand extends SpoCommand {
           });
           propsXml += `<SetProperty Id="${id++}" ObjectPathId="7" Name="${optionKey}"><Parameter Type="Array">${valuesXml}</Parameter></SetProperty><Method Name="Update" Id="${id++}" ObjectPathId="7" />`;
         }
+        else if (['LabelMismatchEmailHelpLink'].indexOf(optionKey) > -1) {
+          // when specifying an empty string, value should be set to null
+          if (optionValue) {
+            propsXml += `<SetProperty Id="${id++}" ObjectPathId="7" Name="${optionKey}"><Parameter Type="String">${optionValue}</Parameter></SetProperty>`;
+          }
+          else {
+            propsXml += `<SetProperty Id="${id++}" ObjectPathId="7" Name="${optionKey}"><Parameter Type="Null">${null}</Parameter></SetProperty>`;
```

Comment:
@milanholemans I am unable to find above code in my forked repository. I think I have lost the code changes from last review during rebase. Is there any way to bring it back, can you please help?

Reply:
Here are the steps I followed:

1. Pulled all changes from `pnp:main` to `main/master` branch of my fork.
2. Cloned forked repository to my new laptop.
3. Used `git rebase` command as you suggested above.
4. There were no merge conflicts shown, so I used `git push --force`.

Reply:
On your local machine, you can use `git reflog` to see what actions you did on your repo. It will contain the git commands you did and all the git commit hashes you can restore. Unfortunately, that's something I cannot view for your repo.
If you are struggling with it, I think I'm able to restore the last 2 commits you made. I'm not sure how many commits this PR contained originally (probably more than 2).